char_creation.php

```diff
@@ -0,0 +1,54 @@
+<html>
+    <head>
+      <a href='project_index.php'>Go back</a>
+    </head>
+    <body>
+        <form action='verify.php' method='POST'>
+            <div>
+               <?php if(isset($_GET['error'])){
+                         echo $_GET['error'];
+                      }  
+                ?>
```

Comment:
Now that you've repeated this chunk of code a couple of times, I'd consider moving it into a function that just displays the error if present - because the function will need to access $_GET directly it won't be unit testable :D